static/templates/stream_settings/stream_types.hbs

```diff
@@ -4,7 +4,7 @@
         {{> ../help_link_widget link="/help/stream-permissions" }}
     </label>
     {{#each stream_privacy_policy_values}}
-        <div class="radio-input-parent">
+        <div class="radio-input-parent settings-radio-input">
```

Comment:
Similarly, this is not an `input` element; the previous class name `radio-input-parent` seems better on this front; maybe overall we want `settings-radio-input-parent` (and for there to be only one class on this element, so we can just rename `radio-input-parent` first).